File: java/Aksesoris.java
import java.util.List;

class Aksesoris extends PetShop {
    private String jenis;
    private String bahan;
    private String warna;

    public Aksesoris(int id, String namaProduk, double hargaProduk, int stokProduk, String jenis, String bahan, String warna) {
        super(id, namaProduk, hargaProduk, stokProduk);
        this.jenis = jenis;
        this.bahan = bahan;
        this.warna = warna;
    }

    public String getJenis() {
        return jenis;
    }

    public String getBahan() {
        return bahan;
    }

    public String getWarna() {
        return warna;
    }

    public static List<String> getColumnHeaders() {
        List<String> headers = PetShop.getColumnHeaders();
        headers.add("JENIS");
        headers.add("BAHAN");
        headers.add("WARNA");
        return headers;
    }

    public List<String> getData() {
        List<String> data = super.getData();
        data.add(jenis);
        data.add(bahan);
        data.add(warna);
        return data;
    }
}

File: python/Aksesoris.py
from PetShop import PetShop

class Aksesoris(PetShop):
    def __init__(self, id, nama, harga, stok, jenis, bahan, warna):
        super().__init__(id, nama, harga, stok)
        self.jenis = jenis
        self.bahan = bahan
        self.warna = warna

    def get_jenis(self):
        return self.jenis

    def get_bahan(self):
        return self.bahan

    def get_warna(self):
        return self.warna

    def get_column_headers(self):
        headers = super().get_column_headers()
        headers.extend(["JENIS", "BAHAN", "WARNA"])
        return headers

    def get_data(self):
        data = super().get_data()
        data.extend([self.jenis, self.bahan, self.warna])
        return data